resources/dex/templates/dex-users-secret.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name:  admin-user
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": "pre-install"
+  labels:
+    "dex-user-config": "true"
+data:
+  email: YWRtaW5Aa3ltYS5jeA==
+  username: YWRtaW4=
+  password: {{ randAlphaNum 12 | b64enc }}
```

Comment:
Is randAlphaNum securely random? Afaics, it ultimately uses https://golang.org/pkg/math/rand/, which is not securely random. Its documentation says:
> For random numbers suitable for security-sensitive work, see the crypto/rand package.

Also, I wonder if 12 characters are enough? Afaics, randAlphaNum draws from an alphabet of 62 symbols. 62^12 is slightly less than 2^72, i.e. we have slightly less than 72 bits of randomness.

How does dex prevent brute-force attacks against passwords? Is there some kind of throttling for username/password authentication? If not, we should aim for at least 256 bits of randomness.

On the other hand: Why the additional round of base64 encoding? It seems that randAlphaNum already uses a subset of the base64 alphabet?